docs/developer/create_application.md

```diff
@@ -0,0 +1,45 @@
+# Create an application
+
+## In DiracX-Web
+
+The applicatins created here will be available for DiracX-Web and for all the extensions.
+
+### Declare the application
+
+In the file `packages/diracx-web-components/src/components/ApplicationList.ts` you can extend the `applicationList` with your new app. You must provide a name (explicit), the component representing the new app and an icon that will appear in the `Add application` menu. You can also give two functions, `setState` and `getState`, to configure the export and import of your app.
+
+### Code the application
+
+The code of your app should be in `packages/diracx-web-components/src/components/<new-app>`. The new app can and should use what already exist in `@dirac-grid/diracx-web-components`. 
+
+In order to be compatible with the share and import buttons, the application must write its state to the session storage at `<appId>_State`. This slot is read from and written to by the corresponding functions.
+
+💡You can look at `JobMonitor` as an example.
+
+## In an extension
+
+### Declare the application
+
+The application can be declared in the file `src/<extension name>/applicationList.ts`. The declaration keeps what is in `diracx-web` and adds application(s) specific to the extension.
+
+### Code the application
+
+The code should be in the folder `src/<extension name>/components/<app name>`. 
+
+To continue, refer to the [Code the application](#code-the-application) section in **DiracX-Web** above.
+
+
+
+## Good practices
+
+- **Code Quality**: Ensure your code is clean, well-documented, and follows best practices. Use tools like [ESLint](https://eslint.org/) and [Prettier](https://prettier.io/) to maintain code quality.
+
+- **Testing**:
+
+  - **Component Testing**: Write tests for your components to ensure they work as expected. Use [Jest](https://jestjs.io/) for unit testing and snapshot testing of your React components.
+  - **Application Testing**: Use [Cypress](https://www.cypress.io/) for end-to-end testing to simulate real user interactions and ensure your application behaves correctly.
+  - **Test Coverage**: Maintain good test coverage to ensure that your critical features are well-protected during updates. Tools like Jest provide [coverage reports](https://jestjs.io/docs/code-coverage) that help you identify untested parts of your code.
+
+- **Accessibility**: Make your application accessible to all users. Use semantic HTML, ARIA attributes, and test your application with different screen sizes and assistive technologies.
+
+By following these practices, you'll ensure that your codebase remains robust, secure, and maintainable.
```

Comment:
Can you move this part to `docs/developer/contribute` (you can merge it with the `Make changes` section) please?